components/LyricsEditor.vue

```diff
@@ -0,0 +1,83 @@
+<script setup lang="ts">
```

Comment:
It currently sends html to the server. Which it should not.

Reply:
Right, forgot to implement the cleanup